src/components/PostCommentCard.jsx

```diff
@@ -0,0 +1,51 @@
+import './CommentCard.css'
+import './PostCommentCard.css'
+import AuthorBubble from './AuthorBubble';
+import { useState } from 'react';
+import { commentOnArticle } from '../../utils/api';
+
+export default function PostCommentCard({ currentUser, article_id, currentComments, setCurrentComments }) {
+    const [newComment, setNewComment] = useState("")
+    const [isLoading, setIsLoading] = useState(false)
+    
+    function handleSubmit(event) {
+        event.preventDefault();
+        if (newComment.length > 0) {
+            setNewComment("posting ...")
+            setIsLoading(true)
+            commentOnArticle(article_id, currentUser.username, newComment)
+            .then((postedComment) => {
+                setNewComment("")
+                // Updating the state only refreshes currentComments.map in ArticleComments.jsx; whereas useEffect reloads the whole page
+                setCurrentComments((currentComments) =>
+                    [postedComment,...currentComments] 
+                )
+                setIsLoading(false)
+            })
+            .catch((err) => {
+                console.log(err.response.data.msg);
+            })
+        }
+    }
+
+    function inactiveButton(event) {
+        event.preventDefault();
+    }
```

Comment:
you could just use the disable attribute on the button to disable is `comment.length < 1`